cloud/cluster/node.go

```diff
@@ -3,7 +3,28 @@ package cluster
 type NodeId string
 
 type Node interface {
+	// A unique node identifier, like 'host:port'
 	Id() NodeId
+
+	// Comparator for sorting.
+	Less(Node) bool
```

Comment:
You had a custom sort by addr then port for wilyNode and I didn't want to remove that ability. I assume you had a reason to not sort by id?